src/fu-util-common.c

```diff
@@ -876,6 +876,130 @@ fu_util_time_to_str (guint64 tmp)
+		/* TRANSLATORS: Is locked and can be unlocked */
+		return _("Device is locked");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_SUPPORTED) {
+		/* TRANSLATORS: Is found in current metadata */
+		return _("Supported on remote server");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_NEEDS_BOOTLOADER) {
+		/* TRANSLATORS: Requires a bootloader mode to be manually enabled by the user */
+		return _("Requires a bootloader");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_NEEDS_REBOOT) {
+		/* TRANSLATORS: Requires a reboot to apply firmware or to reload hardware */
+		return _("Needs a reboot after installation");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_NEEDS_SHUTDOWN) {
+		/* TRANSLATORS: Requires system shutdown to apply firmware */
+		return _("Needs shutdown after installation");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_REPORTED) {
+		/* TRANSLATORS: Has been reported to a metadata server */
+		return _("Reported to remote server");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_NOTIFIED) {
+		/* TRANSLATORS: User has been notified */
+		return _("User has been notified");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_USE_RUNTIME_VERSION) {
+		/* skip */
+		return NULL;
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_INSTALL_PARENT_FIRST) {
+		/* TRANSLATORS: Install composite firmware on the parent before the child */
+		return _("Install to parent device first");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_IS_BOOTLOADER) {
+		/* TRANSLATORS: Is currently in bootloader mode */
+		return _("Is in bootloader mode");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG) {
+		/* TRANSLATORS: The hardware is waiting to be replugged */
+		return _("Hardware is waiting to be replugged");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_IGNORE_VALIDATION) {
+		/* TRANSLATORS: Ignore validation safety checks when flashing this device */
+		return _("Ignore validation safety checks");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_ANOTHER_WRITE_REQUIRED) {
+		/* skip */
+		return NULL;
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_NO_AUTO_INSTANCE_IDS) {
+		/* skip */
+		return NULL;
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_NEEDS_ACTIVATION) {
+		/* TRANSLATORS: Device update needs to be separately activated */
+		return _("Device update needs activation");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_ENSURE_SEMVER) {
+		/* skip */
+		return NULL;
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_HISTORICAL) {
+		/* skip */
+		return NULL;
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_ONLY_SUPPORTED) {
+		/* skip */
+		return NULL;
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_WILL_DISAPPEAR) {
+		/* TRANSLATORS: Device will not return after update completes */
+		return _("Device will not re-appear after update completes");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_CAN_VERIFY) {
+		/* TRANSLATORS: Device supports some form of checksum verification */
+		return _("Cryptographic hash verification is available");
+	}
+	if (device_flag == FWUPD_DEVICE_FLAG_CAN_VERIFY_IMAGE) {
+		/* skip */
```

Comment:
don't we want translations for all of these?

Reply:
Not all of them.  Some of them we definitely don't want to show as it's no value to a user.  This particular one I could go either way (because verify image implies verify).